### ISSUE TITLE
Leave empty lines between CSS selectors

### ISSUE THREAD

**Issue by Jones-S** [victorporof/Sublime-HTMLPrettify#197]
Hi there

I'm not sure if that may be already implemented, but I could not find it in the prefs;
I don't want my empty lines in my scss/css files to be cut out.
I use them for visual separation of the rules and I really would like to keep them.
Is there an option that this:
``` 
.selector1 {
  font-size: 12px;
}

.selector2 {
  font-size: 14px;
}
```

would not end up as:

```
.selector1 {
  font-size: 12px;
}
.selector2 {
  font-size: 14px;
}
```

that would be awesome.
Thanks for your answer.
cheers

**Comment by whazlewo** [comment]
I notice that JS-beautify has this option.
-N, --newline-between-rules        Add a newline between CSS rules

i just don't see a way to enable it in Sublime-HTMLPrettify's options

**Comment by victorporof** [comment]
Last time I checked, --newline-between-rules wasn't in JS-beautify stable, only in a dev branch. As soon as it's available, we'll update.

**Comment by victorporof** [comment]
See https://github.com/victorporof/Sublime-HTMLPrettify/issues/189

**Comment by olets** [comment]
@victorporof I don't where to update this in Sublime-HTMLPrettify so I haven't tested, but the latest js-beautify [v1.6.12](beautify-web/js-beautify/releases/tag/v1.6.12) hopefully fixes this! (see beautify-web/js-beautify#1146)
